src/client/option.go

```diff
@@ -72,3 +72,9 @@ func WithDatumGetFile(datum string) GetFileOption {
 		gf.File.Datum = datum
 	}
 }
+
+func WithOffsetBytes(offset int64) GetFileOption {
```

Comment:
The public API should also be `WithOffset` to match the gRPC message.  Private variables don't matter, but anything user facing does.